query/src/users/auth/auth_mgr.rs

```diff
@@ -64,7 +64,7 @@ impl AuthMgr {
                     None => return Err(ErrorCode::AuthenticateFailure("jwt auth not configured.")),
                 };
                 let user_name = jwt.subject.unwrap();
-                if let Some(create_user) = jwt.custom.create_user {
+                if let Some(create_user) = jwt.custom.ensure_user {
```

Comment:
Some(create_user) => Some(ensure_user) ? q.q

Reply:
Oh no I missed it.